### ISSUE TITLE
Questions about training on cocostuff27

### ISSUE THREAD

**Issue by cwlu201314** [MICV-yonsei/EAGLE#5]
Dear Author,

I have two questions.

First, may I ask if the config file you used to train the model is the same as the one you released? I tried training the model using the file you provided, but the mIoU is only 24.60, and the accuracy is only 53.68. I don't know why.

Second, during the training process, the model's files stop being saved after about 10,000 steps. Is this normal? Does this mean that the model stops improving after around 10,000 steps, or is there something wrong?

**Comment by benearnthof** [comment]
Hi @cwlu201314 I'm currently trying to reproduce the results in this repo and saw your question and that you marked this issue as completed. Did you find a solution? I'm running EAGLE on the cityscapes dataset at the moment and am unable to reproduce the performance in the paper. My test cluster accuracy reaches around 63%, miou consistently stays below 20%. Were you able to reproduce the results on cocostuff? Best regards.

**Comment by cwlu201314** [comment]
Hi,

the answer that author gave me was that you can try to modify the ckpt freq
to save per 10 steps, and it actually helps, but for now I’m unable to
reproduce the performance as well.
The other reason of the low score might due to the inconsistency of the
training process, as the author said, you might need to try more times.

benearnthof ***@***.***>於 2024年9月12日 週四，下午8:55寫道：

**Comment by benearnthof** [comment]
Thank you for the quick reply, I guess this issue is common with Self Supervised Semantic segmentation, STEGO suffers from the same sensitivity to hyperparameters. I'm still curious how exactly the Authors obtained their models, I'd love to use them in downstream applications but if I can't train custom models I may have to use other models.
